doc/spinel-protocol-src/spinel-prop-core.md

```diff
@@ -239,6 +245,59 @@ Unlike most other properties, setting this property to true when the
 value of the property is already true **MUST** fail with a last status
 of `STATUS_ALREADY`.
 
+### PROP 10: PROP_HOST_POWER_STATE {#prop-host-power-state}
+
+* Type: Read-Write
+* Packed-Encoding: `C`
+* Default value: 4
+
+Octets: |        1
+--------|------------------
+Fields: | `HOST_POWER_STATE`
+
+Describes the current power state of the *host*. This property is used
+by the host to inform the NCP when it has changed power states. The
+NCP can then use this state to determine which properties need
+asynchronous updates. Enumeration is encoded as a single unsigned
+byte. These states are defined in similar terms to `PROP_POWER_STATE`
+((#prop-power-state)).
+
+Defined values are:
+
+*   0: `HOST_POWER_STATE_OFFLINE`: Host is physically powered off and
+    cannot be woken by the NCP. All asynchronous commands are
+    squelched.
+*   1: `HOST_POWER_STATE_DEEP_SLEEP`: The host is in a low power state
+    where it can be woken by the NCP but will potentially require more
+    than two seconds to become fully responsive.
```

Comment:
I can't argue with your reasoning, however I feel the process is more bulletproof if we were to require the host to make first contact following a wake from deep sleep.